app/scripts/controllers/detect-tokens.js

```diff
@@ -172,6 +178,20 @@ export default class DetectTokensController {
               iconUrl,
               aggregators,
             } = tokenList[nonZeroTokenAddress];
+
+            this._trackMetaMetricsEvent({
+              event: 'Token Detected',
+              category: EVENT.CATEGORIES.WALLET,
+              sensitiveProperties: {
+                token_symbol: symbol,
+                token_contract_address: address,
+                token_decimal_precision: decimals,
+                source: 'detected',
```

Comment:
We can omit the source property on this one because it will never be a different value